### PR TITLE
Merged with felix and updates for indigo

### PR DESCRIPTION
**Pull request by abubeck**

### CODE REVIEW

**Comment by abubeck** [comment]
@ipa-fxm can you approve this?

**Comment by floweisshardt** [comment]
@abubeck you removed the dependency to pr2_msgs and created a new PowerState message (this is good). But in the source code I can only see that you removed the publishers for pr2_msgs/PowerState but without substitution with the new message. Which component is now publishing the new message?

File: cob_voltage_control/common/src/cob_voltage_control_common.cpp [hunk 1]
```diff
@@ -1,7 +1,5 @@
 //ROS typedefs
 #include "ros/ros.h"
-#include <pr2_msgs/PowerBoardState.h>
```
Comment (floweisshardt) [inline cob_voltage_control/common/src/cob_voltage_control_common.cpp:3]: please use cob_msgs

File: cob_voltage_control/ros/src/cob_voltage_control_ros.cpp [hunk 2]
```diff
@@ -2,8 +2,6 @@
 #include <ros/ros.h>
 
 // ROS message includes
-#include <pr2_msgs/PowerBoardState.h>
```
Comment (floweisshardt) [inline cob_voltage_control/ros/src/cob_voltage_control_ros.cpp:5]: please use cob_msgs

**Comment by fmessmer** [comment]
PR #151 supersedes this PR and also re-adds the PowerBoardState publisher...
Thus, this PR should be empty once #151 is accepted

**Pull request merged**